io.catenax.shopfloor_information.provide_production_tracking/1.0.0/ProvideProductionTrackingData.ttl

```diff
@@ -0,0 +1,103 @@
+##########################################################################################
+# Copyright (c) 2023-2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IOSB & Fraunhofer IWU & Fraunhofer IPA)
+# Copyright (c) 2023-2024 Siemens AG
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:samm:io.catenax.shopfloor_information.provide_production_tracking:1.0.0#>.
+@prefix ext-part: <urn:samm:io.catenax.serial_part:2.0.0#>.
+@prefix ext-header2: <urn:samm:io.catenax.shared.message_header:2.0.0#>.
+
+:ProvideProductionTrackingData a samm:Aspect;
+    samm:preferredName "ProvideProductionTrackingData"@en;
+    samm:properties (:processStepIdentifierList ext-part:localIdentifiers ext-header2:header ext-header2:version);
+    samm:operations ();
+    samm:events ().
+
+:processStepIdentifierList a samm:Property;
+    samm:preferredName "processStepIdentifierList"@en;
```

Comment:
preferredName value should be in plain English form without any camel case.
please correct all other places also.